CenterForOpenScience/ember-osf-web
# [ENG-2315] Make text in finalize-registration-modal conditional on moderation

- Ticket: [ENG-2315]
- Feature flag: n/a

## Purpose
- Update the text shown in the `Almost done...` modal at the end of draft registration submission to reduce confusion for users
  - this update is assuming that the registration is based on a project. More language will be added for no-project based registrations in an upcoming PR for [ENG-2325]
- Conditionally show that their registration is subject to moderation

## Summary of Changes
- Update the finalize-registration-modal to show different text depending on if the provider has reviewsWorkflow

For moderated providers:
![Screen Shot 2021-01-21 at 10 32 10 AM](https://user-images.githubusercontent.com/51409893/105372748-ec8fb480-5bd3-11eb-8fe5-80f1084eb48d.png)


For non-moderated providers:
![Screen Shot 2021-01-21 at 10 29 11 AM](https://user-images.githubusercontent.com/51409893/105372355-81de7900-5bd3-11eb-8954-3850a6e4a4fb.png)



## Side Effects
- I've removed a lot of the whitespace that was in the translation file. I don't think this should make any difference though.

## QA Notes
- Please double check that for non-moderated registration providers (ie: OSF Registries) does not show last bullet point "A moderator must review and approve your registration..." and that moderated providers show this message. I think some providers on staging2 do not have moderation enabled, so if this message is not showing up unexpectedly, it may be wise to check there API/admin app if `reviewsWorkflow` is set to `pre-moderation`

[ENG-2315]: https://openscience.atlassian.net/browse/ENG-2315

[ENG-2325]: https://openscience.atlassian.net/browse/ENG-2325
